librz/util/ht/ht_sp.c

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: 2024 pelijah
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
You want this under `BSD-3`? We use by default `LGPL-3.0-only`. But just a question.

Reply:
All other files are under BSD-3